tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp

```diff
@@ -648,6 +663,22 @@ namespace SimulationTests
 			TestResult();
 			CheckBandLinearSolverDisabled();
         }
+
+		[TestAttribute]
+		void should_calculate_comparison_threshold()
+		{
+			//get absolute tolerance used for calculation (might differ from input absolute tolerance)
+			const double AbsTol = sut->UsedAbsoluteTolerance;
+
+			//expected threshold for ode variables
+			const double threshold = 10.0 * AbsTol;
+
+			BDDExtensions::ShouldBeEqualTo(_y1->GetComparisonThreshold(), threshold * _scaleFactor);
+			BDDExtensions::ShouldBeEqualTo(_y2->GetComparisonThreshold(), threshold * _scaleFactor);
+
+			//scale factor is ignored for constant variables!
+			BDDExtensions::ShouldBeEqualTo(_y3->GetComparisonThreshold(), threshold);
```

Comment:
Constant variable does not change along time and is eliminated from the ODE system
(it's just filled with is initial value).
E.g. all non-floating enzymes, transporters etc. will be const.

Actually AbsTol does not matter for such variables at all: All values are equal to the initial value of the variable defined in PK-Sim or MoBi